hierarc/Likelihood/cosmo_likelihood.py

```diff
@@ -45,6 +46,8 @@ def __init__(self, kwargs_likelihood_list, cosmology, kwargs_bounds, sne_likelih
         :param lambda_ifu_distribution: string, distribution function of the lambda_ifu parameter
         :param alpha_lambda_sampling: bool, if True samples a parameter alpha_lambda, which scales lambda_mst linearly
          according to a predefined quantity of the lens
+        :param beta_lambda_sampling: bool, if True samples a parameter beta_lambda, which scales lambda_mst linearly
+         according to a predefined quantity of the lens
```

Comment:
can you describe a bit in more detail how and where this 'pre-defined quantity' gets described?